core-bundle/src/Resources/config/listener.yml

```diff
@@ -88,6 +88,28 @@ services:
         tags:
             - { name: contao.hook, hook: loadDataContainer }
 
+    contao.listener.data_container.custom_element_template_options_listener:
+        class: Contao\CoreBundle\EventListener\DataContainer\CustomTemplateOptionsListener
+        arguments:
+            - '@contao.framework'
+            - '@request_stack'
+            - '%contao.content_element.templates%'
+            - '%contao.content_element_template_prefix%'
+            - '%contao.content_proxy%'
+        tags:
+            - { name: contao.callback, table: tl_content, target: fields.customTpl.options }
+
+    contao.listener.data_container.custom_module_template_options_listener:
+        class: Contao\CoreBundle\EventListener\DataContainer\CustomTemplateOptionsListener
+        arguments:
+            - '@contao.framework'
+            - '@request_stack'
+            - '%contao.frontend_module.templates%'
+            - '%contao.frontend_module_template_prefix%'
```

Comment:
I don't think these need to be parameters, noone needs to change them, we can just hardcode it here.